.ci/config/stage-types.yaml

```
@@ -62,7 +62,7 @@ instance-migration:
     images: ['maven:3.9.7-eclipse-temurin-11', 'postgres:17.0']
 old-engine:
   directory: 'qa'
-  command: 'verify -Pold-engine,'
+  command: 'verify -Pold-engine,exclude-post-jdk15-tests,'
```

🔧 `instance-migration` uses jdk 11 so this is no longer needed:
```suggestion
  command: 'verify -Pold-engine,'
```

---

Indeed, done ✅